### ISSUE TITLE
[Tuning] Zscaler.ZIA.PasswordExpiration - Wrong alert title mentions SAML instead of password

### ISSUE THREAD

**Issue by zestysoft** [panther-labs/panther-analysis#1878]
## Rule ID

Zscaler.ZIA.PasswordExpiration

## Description

The `title()` function returns an incorrect alert title. It says "SAML configuration was changed" but this rule detects **password expiration** changes, not SAML configuration changes.

```python
def title(event):
    return (
        f"[Zscaler.ZIA]: SAML configuration was changed by admin with id "
        f"[{event.deep_get('event', 'adminid', default='<ADMIN_ID_NOT_FOUND>')}]"
    )
```

This appears to be a copy-paste error from `zia_trust_modification.py`.

When this rule triggers, security analysts will see misleading alert titles that reference SAML instead of password expiration, causing confusion during incident response.

**Suggested Fix:** Update the title to accurately reflect the detection:

```python
def title(event):
    return (
        f"[Zscaler.ZIA]: Password expiration setting was changed by admin with id "
        f"[{event.deep_get('event', 'adminid', default='<ADMIN_ID_NOT_FOUND>')}]"
    )
```
